### DISCUSSION TITLE
How to login Root (aka Super or Admin) tenant after changing to use DomainTenantResolver

### DISCUSSION THREAD

**Discussion by nguyenhmtriet** [abpframework/abp#13891] [Q&A]
Hi guys,

My project used to default tenant resolver which were QueryString/Cookie. These will enable **Switch tenant** button for us to change tenant on Login page. After a while, my team decided to change using subdomain for each tenant (e.g: **https://tenant-xx.mydomain.com**). It works perfectly for all of tenants.

However, I couldn't login to root account without typing subdomain (e.g: https://mydomain.com). According to this [DomainTenantResolver](https://github.com/abpframework/abp-samples/tree/master/DomainTenantResolver) sample, I try to import as much as possible modules. But I didn't know whether I missed some modules which were required to enable Admin login.

**Comment by maliming** [comment]
> `subdomain (e.g: https://mydomain.com/)` will be resolved to Host.

Can you check the logs?

**Reply by nguyenhmtriet** [reply]
> > `subdomain (e.g: https://mydomain.com/)` will be resolved to Host.
> 
> Can you check the logs?

yes, I can, here the logs are

![image](https://user-images.githubusercontent.com/24971905/187847311-ac567dc4-f58e-4110-b0dc-c861edfaabc2.png)

**Reply by nguyenhmtriet** [reply]
I wonder whether this is helpful for you to clarify it.

**Reply by maliming** [reply]
hi

There are just `info` logs, I didn't see any issues.


You can copy and add a new `DomainTenantResolveContributor` and try to debug it.

https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.AspNetCore.MultiTenancy/Volo/Abp/AspNetCore/MultiTenancy/DomainTenantResolveContributor.cs

**Reply by nguyenhmtriet** [reply]
Hi,

It's all I have, because on my eyes, there is no ERR flag at all. But there is a line to say invalid password for user.

I have a question here. Should I name for root tenant. As I know It's always the empty tenant name for root user. The reason due to we changed to use DomainTenantResolveContributor I think.
